quiz/question.go

```diff
@@ -0,0 +1,38 @@
+package quiz
+
+/*
+	Question represents each the individual elements of a quiz. It stores the users answer and whether that answer
+	is correct.
+*/
+type Question struct {
+	QuestionText string
+	Answer       string
+	UserAnswer   string
+	Correct      *bool
+}
+
+/*
+	Creates a new question using the given questionText and answer.
+	UserAnswer is initially stored as "<empty>" and the Correct pointer is initially stored as nil.
+	These values are updated when a quiz is taken.
+*/
```

Comment:
These are java style comments.
Go comments are done using line starting with `//` which are then picked up into godocs natively.

Also exported fields, variables, funcs, methods, types, etc. will usualy start with the name of the thing (e.g. `\\ NewQuestion`). 

The linter should be catching this so you may have a bigger problem.